example/e2e/constants.ts

```diff
@@ -135,4 +135,14 @@ export enum UserFeedback {
   INVALID_CITY_LABEL = 'Please enter a valid city',
   TRANSACTION_PREVENTED = 'The recommendation server has prevented this transaction.',
   RECOMMENDATION_ERROR = 'There was an error when retrieving the recommendation response.',
+  IDEAL_ERROR = 'The request has failed or responded without data.',
+  IDEAL_ERROR_IOS = 'The transaction was cancelled by the user.',
```

Comment:
Just these two errors are not iDeal-specific, can also occur in other transactions. Wouldn't their names be misleading?

Reply:
You're right - pushed an commit with more generic naming